recipe/build_base.bat

```diff
@@ -1,10 +1,8 @@
 @echo on
 set "PYO3_PYTHON=%PYTHON%"
 
-set MATURIN_SETUP_ARGS=--no-default-features --features=native-tls
-
 set "CMAKE_GENERATOR=NMake Makefiles"
-maturin build -v --jobs 1 --release --strip --manylinux off --interpreter=%PYTHON%
+maturin build -v --jobs 1 --release --strip --manylinux off --interpreter=%PYTHON% --no-default-features --features=native-tls
```

Comment:
shouldn't we use `MATURIN_PEP517_ARGS` here as well?

Reply:
My assumption is that the `MATURIN_PEP517_ARGS` injects these arguments into this command when doing `pip install`.

Reply:
ah right, we do `maturin build` here 👍🏻